db/seeds.rb

```diff
@@ -5,3 +5,6 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+image_files = Rails.root.join(Settings.IMAGE_PATH, '*.jpg')
```

Comment:
Ok cool. Created a ticket here for this: https://github.com/sul-cidr/histonets/issues/27